data/sgid-base-map-services-arcmap/index.md

```diff
@@ -27,14 +27,13 @@ Instructions for using Discover services in ArcMap 10.x:
 1. Navigate to the newly added `utah imagery – WMTS on discover.agrc.utah.gov` connection and **double click** to connect.
   - You can rename the connection after it has been added
 1. You should see a list of imagery and base map services (explained below):
-  - `utah` – 6-inch statewide natural color aerial photography collected within the last 3 years (licensed)
-  - `utah_2012archive` – Aerial photography updated in `utah` service and archived by acquisition year (licensed)
-  - `utah_2013archive` – Aerial photography updated in `utah` service and archived by acquisition year (licensed)
+  - `utah` – 6-inch statewide natural color aerial photography collected within the last 3 years **(licensed)**
+  - `utah_2012archive` – Aerial photography updated in `utah` service and archived by acquisition year **(licensed)**
+  - `utah_2013archive` – Aerial photography updated in `utah` service and archived by acquisition year **(licensed)**
   - `doq_1990’s_bw` – 1-meter statewide black & white aerial photography collected in 1990’s (coming soon)
   - `hro_2012_rgb` – 12.5-cm natural color aerial photography collected in 2012 of the Wasatch Front
-  - `naip_2009_rgb` – 1-meter statewide natural color aerial photography collected in 2009
-  - `naip_2011_nrg` – 1-meter statewide color infrared aerial photography collected in 2011
-  - `naip_2011_rgb` – 1-meter statewide natural color aerial photography collected in 2011
+  - `naip_*year*_rgb` – 1-meter statewide natural color aerial photography
```

Comment:
nope but should look like this
naip_*year*_nrg

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

On Thu, Mar 31, 2016 at 1:18 PM, steveoh <notifications@github.com> wrote:

> In data/sgid-base-map-services-arcmap/index.md
> <https://github.com/agrc/agrc.github.io/pull/79#discussion_r58111961>:
>
> >    - `doq_1990’s_bw` – 1-meter statewide black & white aerial photography collected in 1990’s (coming soon)
> >    - `hro_2012_rgb` – 12.5-cm natural color aerial photography collected in 2012 of the Wasatch Front
> > -  - `naip_2009_rgb` – 1-meter statewide natural color aerial photography collected in 2009
> > -  - `naip_2011_nrg` – 1-meter statewide color infrared aerial photography collected in 2011
> > -  - `naip_2011_rgb` – 1-meter statewide natural color aerial photography collected in 2011
> > +  - `naip_*year*_rgb` – 1-meter statewide natural color aerial photography
>
> i don't think this is working like you expect
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/agrc/agrc.github.io/pull/79/files/1cd416cf8071c4ad98b2879ff65d5dfd0119ba5e#r58111961>
>